frontend/dashboard/components/OrgRepoList/OrgReposList.tsx

```diff
@@ -10,30 +10,37 @@ import { useSelectedContext } from 'dashboard/hooks/useSelectedContext';
 import { Heading } from '@digdir/design-system-react';
 import { DATAGRID_DEFAULT_PAGE_SIZE } from 'dashboard/constants';
 import { useAugmentReposWithStarred } from 'dashboard/hooks/useAugmentReposWithStarred';
-import { useStarredReposQuery } from 'dashboard/hooks/queries';
+import { useSearchReposQuery, useStarredReposQuery } from 'dashboard/hooks/queries';
 
 type OrgReposListProps = {
   user: User;
   organizations: Organization[];
 };
 export const OrgReposList = ({ user, organizations }: OrgReposListProps) => {
-  const { data: starredRepos = [], isPending: areStarredReposPending } = useStarredReposQuery();
-  const selectedContext = useSelectedContext();
   const { t } = useTranslation();
+  const selectedContext = useSelectedContext();
   const uid = getUidFilter({ selectedContext, userId: user.id, organizations });
 
   const {
-    searchResults,
-    isLoadingSearchResults,
-    sortModel,
+    searchResults: repoResults,
+    isLoadingSearchResults: areReposPending,
     pageSize,
-    setSortModel,
+    pageNumber,
     setPageNumber,
     setPageSize,
+    onSortClick,
   } = useReposSearch({ uid: uid as number, defaultPageSize: DATAGRID_DEFAULT_PAGE_SIZE });
 
+  const { data: dataModelsResults, isPending: areDataModelsPending } = useSearchReposQuery({
+    uid: uid as number,
+    keyword: '-datamodels',
+    page: 0,
+  });
+  const totalRows = repoResults?.totalCount - dataModelsResults?.totalCount ?? 0;
```

Comment:
If we adapt the mapQueryParams to include the keyword and maybe advance it to a `not-keyword`?? Or something 😅  Then we could use the useReposSearch directly and avoid doing a new query to the useSearchReposQuery

Reply:
I'm not sure that I understand, but I don't think the Gitea API supports excluding-keywords, hence why I'm making a second API-call to the backend to separate the data models from the non-datamodels. In the network console, it seems like React Query caches this query and reuses it in `DataModelsRepoList` anyway, so there should not be any performance impact by doing a second query here (if my thinking is correct).

Another solution could be to filter the data models from the non-datamodels on the client side with `.filter()` and get the length of that array. What do you think? 😊